site/public/js/calendar.js

```diff
@@ -0,0 +1,318 @@
+// List of names of months in English
+const monthNames = ['December', 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+const monthNamesShort = ['Dec', 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sept', 'Oct', 'Nov', 'Dev'];
+
+/**
+ * Gets the previous month of a given month
+ * @param month : int the current month (1 as January and 12 as December)
+ * @param year : int the current year
+ * @returns {number[]} : array<int> {previous_month, year_of_previous_month}
+ */
+// eslint-disable-next-line no-unused-vars
```

Comment:
You should be doing `/* exported prevMonth */` at the top of this file per the note at https://submitty.org/developer/coding_style_guide/javascript#style-rules and not explicitly disabling this rule. This applies to the other instances of `// eslint-disable-next-line` you have in this file for functions.